### ISSUE TITLE
No link to current WashU Epigenome Browser

### ISSUE THREAD

**Issue by jaflynn5** [debugpoint136/virus-browser#21]
In the "Help" drop-down menu, there is a link to the 'old browser' (which may seem confusing to people since it's referring to the old epigenome browser and not an old virus browser).  Can this link be replaced with one to the WashU Epigenome Browser (current version)?

**Comment by debugpoint136** [comment]
Resolved by `virusBrowserMode=1` option

**Comment by debugpoint136** [comment]
@jaflynn5 Please confirm if still an issue

**Comment by jaflynn5** [comment]
Yes, the "Help" menu no longer displays. Should we still provide a link to the epigenome browser though? Not sure where this would be most appropriate. Maybe at the bottom of the landing page in the dark gray box?
